yarn.lock

```diff
@@ -4,241 +4,269 @@
 
 "@cspotcode/source-map-support@^0.8.0":
   version "0.8.1"
-  resolved "https://registry.yarnpkg.com/@cspotcode/source-map-support/-/source-map-support-0.8.1.tgz#00629c35a688e05a88b1cda684fb9d5e73f000a1"
+  resolved "https://registry.npmjs.org/@cspotcode/source-map-support/-/source-map-support-0.8.1.tgz"
```

Comment:
You need to revert this. Normal yarn uses yarnnpkg, and your does not. So the next person (or any auto script) will have problems.